cosmovisor/upgrade.go

```diff
@@ -36,11 +37,11 @@ func DoUpgrade(cfg *Config, info upgradetypes.Plan) error {
 	}
 
 	// If not there, then we try to download it... maybe
-	Logger.Info().Msg("No upgrade binary found, beginning to download it")
+	logger.Info().Msg("No upgrade binary found, beginning to download it")
```

Comment:
```suggestion
	logger.Info().Msg("no upgrade binary found, beginning to download it")
```

We should be consistent with the punctuation of our logs here. Typically, we don't capitalize, but it's not a huge dea.

Reply:
I agree